packages/strapi-admin/admin/src/components/Users/LoginModalSection/index.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ModalSection } from 'strapi-helper-plugin';
+import { Padded } from '@buffetjs/core';
+import { Row } from 'reactstrap';
+import { useIntl } from 'react-intl';
+import loginSettingsForm from 'ee_else_ce/components/Users/ModalCreateBody/utils/loginSettingsForm';
+
+import Input from '../../SizedInput';
+import Wrapper from '../ModalCreateBody/Wrapper';
+
+const LoginModalSection = ({ isDisabled, modifiedData, onChange, formErrors }) => {
```

Comment:
I don't like the name of this component, it makes me think a user can login in the modal, can we name it, `RoleSettingsModalSection` or something similar? 
What do you think?

Reply:
You are right 👍🏽